<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenomAMf</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QGroupBox {
    border: 1px solid gray;
    border-radius: 3px;
    margin-top: 1ex; 
	padding: 9px;
 }

 QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
 }*/

QDockWidget::title {
    text-align: left; /* align the text to the left */
    padding-left: 6px;
    padding-top: 4px;
 }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="loadSequencesAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuA_n_lisis">
    <property name="title">
     <string>A&amp;n치lisis</string>
    </property>
    <addaction name="makeMultifractalAnalisysAction"/>
    <addaction name="actionCorrelaci_n"/>
   </widget>
   <widget class="QMenu" name="menu_Herramientas">
    <property name="title">
     <string>&amp;Herramientas</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuJuego_del_Caos">
    <property name="title">
     <string>&amp;Juego del Caos</string>
    </property>
    <addaction name="makeCgrAction"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>&amp;Test</string>
    </property>
    <addaction name="testAction"/>
   </widget>
   <widget class="QMenu" name="menu_Ventana">
    <property name="title">
     <string>&amp;Ventana</string>
    </property>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menuJuego_del_Caos"/>
   <addaction name="menuA_n_lisis"/>
   <addaction name="menu_Herramientas"/>
   <addaction name="menu_Ventana"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Explorador de Objetos</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="margin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
      </layout>
     </item>
    </layout>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </widget>
  </widget>
  <action name="loadSequencesAction">
   <property name="text">
    <string>&amp;Adicionar Secuencias</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="toolTip">
    <string>Salir de la aplicaci칩n</string>
   </property>
  </action>
  <action name="makeCgrAction">
   <property name="text">
    <string>Realizar &amp;Representaci칩n ...</string>
   </property>
  </action>
  <action name="makeMultifractalAnalisysAction">
   <property name="text">
    <string>&amp;Multifractal ...</string>
   </property>
  </action>
  <action name="actionCorrelaci_n">
   <property name="text">
    <string>&amp;Correlaci칩n ...</string>
   </property>
  </action>
  <action name="testAction">
   <property name="text">
    <string>&amp;Test Action</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
